flink-autoscaler/src/main/java/org/apache/flink/autoscaler/ScalingExecutor.java

```diff
@@ -264,4 +262,21 @@ private static Map<String, String> getVertexParallelismOverrides(
                 });
         return overrides;
     }
+
+    private boolean blockScalingExecution(
```

Comment:
We should rename this to `triggerScalingEvent` or `checkIfBlockedAndTriggerScalingEvent` or add javadocs or something that suggests that the scaling event is triggered inside

Reply:
Renamed to `checkIfBlockedAndTriggerScalingEvent`